src/GitHub.UI.Reactive/Controls/Validation/ValidationMessage.cs

```diff
@@ -33,7 +34,7 @@ public ValidationMessage()
                 .Do(CreateBinding)
                 .Select(control =>
                     Observable.Merge(
-                        this.WhenAnyValue(x => x.ShowError),
+                        this.WhenAnyValue(x => x.ShowError).Where(x => userHasInteracted),
```

Comment:
Yes, `WhenAnyValue` is triggered immediately. The way of not making it triggered is to use `.WhenAny(...).Skip(1)` but that didn't work here because you get a load of different values pushed, which start off `false` and then become `true`. 

So rather than deal with this, and because the intention is to only show user errors after user interaction, i decided that the best way to do it was explicitly by setting a flag on user interaction.